src/rp2350/pico_platform/include/pico/platform.h

```diff
@@ -23,13 +23,14 @@
 #include "pico/platform/compiler.h"
 #include "pico/platform/sections.h"
 #include "pico/platform/panic.h"
+#include "pico/platform/common.h"
 #include "hardware/regs/addressmap.h"
 #include "hardware/regs/sio.h"
 #ifdef __riscv
 #include "hardware/regs/rvcsr.h"
 #endif
 
-// PICO_CONFIG: PICO_RP2350A, Whether the current board has an RP2350 in an A (30 GPIO) package, type=bool, default=Usually provided via board header, group=pico_platform
+// PICO_CONFIG: PICO_RP2350A, Whether the current board has an RP2350 in an A (30 GPIO) package - set to 0 for RP2350in a B (48 GPIO) package, type=bool, default=Usually provided via board header, group=pico_platform
```

Comment:
nit: missing space in "RP2350in"

Reply:
Just to clarify - towards the end of the line it says "set to 0 for RP2350in a B (48 GPIO) package" which ought to be "set to 0 for RP2350 in a B (48 GPIO) package"